package.json

```diff
@@ -22,6 +25,10 @@
     "node-fetch": "^2.6.1"
   },
   "devDependencies": {
+    "@types/dotenv": "^8.2.0",
+    "@types/express": "^4.17.9",
+    "@types/node": "^14.14.10",
+    "@types/node-fetch": "^2.5.7",
     "typescript": "^4.1.2"
   }
 }
```

Comment:
I've apperently forgot about creating file `.eslintrc`
```
{
  "root": true,
  "parser": "@typescript-eslint/parser",
  "plugins": [
    "@typescript-eslint"
  ],
  "extends": [
    "eslint:recommended",
    "plugin:@typescript-eslint/eslint-recommended",
    "plugin:@typescript-eslint/recommended"
  ],
  "rules": {
    "@typescript-eslint/no-explicit-any": 2
  }
}
```

Also  looks like you didn't install the packages I asked for.
`npm install --save-dev eslint @typescript-eslint/parser @typescript-eslint/eslint-plugin patch-package`

they're not listen in package.json here

Reply:
After you'll introduce these things, linter will find issues with your code
`npm run lint`


```
/home/mkurowski/OSS/houston/src/api/n2yo/index.ts
  7:5   warning  Missing return type on function                                               @typescript-eslint/explicit-module-boundary-types
  8:15  error    Type string trivially inferred from a string literal, remove type annotation  @typescript-eslint/no-inferrable-types

/home/mkurowski/OSS/houston/src/index.ts
  6:5  error  'app' is never reassigned. Use 'const' instead  prefer-const
```
Get rid of them